orchestra/static/orchestra/common/components/checklist/js/directives.js

```diff
@@ -0,0 +1,243 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('orchestra.common.components.directives')
+    .directive('orchestraChecklist', ['$http', '$timeout', '$compile', 'orchestraService', orchestraChecklist]);
+
+  function orchestraChecklist($http, $timeout, $compile, orchestraService) {
+    return {
+      restrict: 'E',
+      scope: {
+        'data': '=',
+        'author': '=?',
+        'imagePrefix': '=?',
+      },
+      link: function(scope, el, attr) {
+        var sampleItem = {
+          'title': 'This is a sample checklist item. ' +
+                   'Double-click on the text to edit, ' +
+                   'or click to the right to expand.',
+          'comments': [
```

Comment:
similarly: can we prime folks to view this as a conversation rather than a single explanatory comment?